content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -20,15 +20,21 @@ You will need to have the following tools installed:
 - `cosign` ([install guide](https://docs.sigstore.dev/cosign/installation/))
 - `curl` (often provided by your operating system)
 
+{{< note >}}
+`COSIGN_EXPERIMENTAL=1` is no longer required to have identity-based ("keyless") signing and transparency.
+
+For additional information please read the Cosign 2.0 [blog post](https://blog.sigstore.dev/cosign-2-0-released/).
+{{< /note >}}
```

Comment:
Can you rephrase this to be [timeless](https://developers.google.com/style/timeless-documentation)?

For example, start with: “Previous versions of Cosign required that you enabled the `COSIGN_EXPERIMENTAL`&hellip;”